alexandria/core/serializers.py

```diff
@@ -15,6 +16,7 @@
 from . import models
 
 log = logging.getLogger(__name__)
+get_user_and_group = import_string(settings.ALEXANDRIA_GET_USER_AND_GROUP_FUNCTION)
```

Comment:
I'd move it into a method of the `BaseSerializer`. If you have it here, it's hard to change during tests, as it won't be re-evaluated if you change the setting. Or keep it just inline where it's being called instead.